gameSource/languages/help_English

```diff
@@ -11,13 +11,17 @@
 title$MY PHRASEBOOK$
 IF I SAY...
 space$0.25f
-sub$'NO BB'$
+sub$'I AM XXX.'$
+I WILL GIVE MYSELF A NAME.
+sub$'YOU ARE XXX.'$
+I WILL NAME SOMEONE.
+sub$'LET ME BE INFERTILE.'$
 I WILL NOT HAVE BABIES.
-sub$'GIMME BB'$
```

Comment:
How come you've replaced the 'GIMMIE BB' part? This is still a valid phrase.

Reply:
GIMME BB is very informal and menus usually are not written in non-standard language.
New players can learn this phrase later interacting with the community.